src/vkex/Device.cpp

```diff
@@ -399,6 +417,40 @@ vkex::Result CDevice::InitializeQueueRequests()
   return vkex::Result::Success;
 }
 
+void CDevice::InitializeExtensionFeatures()
+{
+  m_create_info.app_p_next = m_create_info.p_next;
```

Comment:
Whats the `app_p_next` for exactly? Seems redundant if the existing `p_next` is treated properly.

Reply:
In my head, I kinda wanted to keep the original application `p_next` around in case I wanted to check it. But now...I don't think that's really important. Thanks for pointing it out, I'll get rid of it.

Reply:
+1